AutomatedTesting/Gem/PythonTests/assetpipeline/fbx_tests/scene_scripting_tests.py

```diff
@@ -0,0 +1,67 @@
+#
+# Copyright (c) Contributors to the Open 3D Engine Project.
+# For complete copyright and license terms please see the LICENSE at the root of this distribution.
+#
+# SPDX-License-Identifier: Apache-2.0 OR MIT
+#
+#
+
+
+import pytest, logging, os
+
+# Import fixtures
+from ..ap_fixtures.asset_processor_fixture import asset_processor as asset_processor
+from ..ap_fixtures.ap_setup_fixture import ap_setup_fixture as ap_setup_fixture
+
+# Import LyShared
+import ly_test_tools.o3de.pipeline_utils as utils
+
+# Use the following logging pattern to hook all test logging together:
+logger = logging.getLogger(__name__)
+
+@pytest.fixture
+def local_resources(request, workspace, ap_setup_fixture):
+    ap_setup_fixture["tests_dir"] = os.path.dirname(os.path.realpath(__file__))
+
+@pytest.mark.usefixtures("asset_processor")
+@pytest.mark.usefixtures("ap_setup_fixture")
+@pytest.mark.usefixtures("local_resources")
+@pytest.mark.parametrize("project", ["AutomatedTesting"])
+@pytest.mark.SUITE_sandbox
+class TestsSceneScripting(object):
+
+    @property
+    def asset_processor_extra_params(self):
+        return [
+        # Disabling Atom assets disables most products, using the debugOutput flag ensures one product is output.
+        "--debugOutput",
+        # By default, if job priorities are equal, jobs run in an arbitrary order. This makes sure
+        # jobs are run by sorting on the database source name, so they run in the same order each time
+        # when this test is run.
+        "--sortJobsByDBSourceName",
+        # Disabling Atom products means this asset won't need a lot of source dependencies to be processed,
+        # keeping the scope of this test down.
+        "--regset=\"/O3DE/SceneAPI/AssetImporter/SkipAtomOutput=true\"",
+        # The bug this regression test happened when the same builder processed FBX files with and without Python.
+        # This flag ensures that only one builder is launched, so that situation can be replicated.
+        "--regset=\"/Amazon/AssetProcessor/Settings/Jobs/maxJobs=1\""]
```

Comment:
Do you need this flag here? This test seems like it would be fine if more builders were available

Reply:
the test wants exactly one asset to be processed that will pick up the script to export the default_script product... do you see harm in this flag?